src/Tizen.NUI.Components/Controls/Menu.cs

```diff
@@ -93,9 +131,9 @@ public IEnumerable<MenuItem> Items
                 {
                     foreach (var oldItem in menuItems)
                     {
-                        if (scrollableBase.Children?.Contains(oldItem) == true)
+                        if (content.Children?.Contains(oldItem) == true)
                         {
-                            scrollableBase.Children.Remove(oldItem);
+                            content.Children.Remove(oldItem);
```

Comment:
You should call `content.Remove(oldItem)` instead `content.Children.Remove` because `content.Children` is not Observerable collection. so it does not anything when children was removed

And, you do not need to check `Contains` as my opinions, 
I think, just call `Unparen() `is enough

``` c#
   oldItem.Unparent();
```

Reply:
Thank you for your comment.
I have changed as follows based on your comment.
```
if (content.Children?.Contains(oldItem) == true)
{
    content.Remove(oldItem);
}
```

View.Remove(child) calls functions related to Remove although View does not contain the child.
(It only shows Tizen.Log.Error messages if View does not contain the child.)
So I think that checking if content contains oldItem may prevent unnecessary calls related to Remove.

I am not sure what app developer does to Items.
e.g. app developer may add Items to other View right before setting a new Items to Menu.
So I think that content.Remove() is safer than oldItem.Unparent().